schema-admin.graphql

```diff
@@ -487,6 +487,97 @@ input CreateApprovedCuratedCorpusItemInput {
     scheduledSurfaceGuid: ID
 }
 
+"""
+Input data for loading an Approved Item via an automated process and optionally scheduling
+this item to appear on a Scheduled Surface.
+"""
+input LoadApprovedCuratedCorpusItemInput {
+    """
+    The GUID of the corresponding Prospect ID. Will be empty for manually added items.
+    """
+    prospectId: ID
```

Comment:
the prospectId will be passed as `null` for these mutation, so do we need this in the input?

Reply:
I think it should be okay since it's not a required field, we can pass it as `null`